src/qml/pages/ChatPage.qml

```diff
@@ -75,6 +75,64 @@ Page {
                     color: Theme.secondaryTextColor
                 }
             }
+
+            ImageButton {
+                image: "More"
+                onClicked: {
+                    if (appState.isGroupChat) {
+                        if (appState.isAdmin) {
+                            groupChatAdminContextMenu.currentIndex = -1
```

Comment:
I guess context menu is opened with selected item and you reset selection.
Don't set currentIndex before every opening, move `currentIndex = -1` inside the ContextMenu component

Reply:
I copied you code from. Okay, I will fix it